components/Heroes/heroes.css

```diff
@@ -0,0 +1,79 @@
+.Header {
```

Comment:
Components and anything related to components directly (such as css), we have tried to capitalize. Other things can be lower case. I don't know if we have done that in every case though.